cmd/agent/daemon/state/netflow_pipeline.go

```diff
@@ -54,7 +60,7 @@ func (c *Controller) runNetflowPipeline(ctx context.Context) error {
 	if err != nil {
 		c.log.Errorf("getting cluster info: %v", err)
 	} else {
-		c.log.Infof("fetched cluster info, pod_cidr=%s, cluster_cidr=%s", c.clusterInfo.podCidr, c.clusterInfo.serviceCidr)
+		c.log.Infof("fetched cluster info, pod_cidr=%s, service_cidr=%s", c.clusterInfo.podCidr, c.clusterInfo.serviceCidr)
```

Comment:
Could you also add some timeout for `getClusterInfo` call, like 20 seconds or so. It's already optional but I missed that it uses global context and will never finish in case of some error or bug.